Sources/UIColor+Sweet.swift

```diff
@@ -21,6 +21,17 @@ public extension UIColor {
         }
     }
 
+    /// Convenience initializers for RGB colors.
+    ///
+    /// - Parameters:
+    ///   - red: The red part.
+    ///   - green: The green part.
+    ///   - blue: The blue part.
+    ///   - alpha: The alpha part.
+    public convenience init(r red: Double, g green: Double, b blue: Double, a alpha: Double = 100) {
```

Comment:
This one uses percentages? I thought from the implementation that it ranged from 0-255, like CSS etc. 🤔 No good ideas regarding the method name so far, but documentation could clarify, including by adding the range information for the parameters. 😄